src/main/java/dev/waterdog/waterdogpe/command/Command.java

```diff
@@ -87,6 +91,10 @@ public CommandData craftNetwork() {
         CommandParamData[][] parameterData = new CommandParamData[][]{{
                 new CommandParamData(this.name, true, null, CommandParam.TEXT, null, Collections.emptyList())
         }};
-        return new CommandData(this.name, this.getDescription(), Collections.emptyList(), (byte) 0, new CommandEnumData(this.name, getAliases(), false), parameterData);
+        List<String> aliases = new ArrayList<>(Arrays.asList(getAliases()));
```

Comment:
Since there are not supposed to be any duplicates in here, you could use a `Set` instead of a `List` and then simply call `add` which does the contains part all by itself.

Reply:
Yeah, you are right.